packages/accounts-controller/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@metamask/accounts-controller",
+  "version": "1.0.0",
+  "description": "Manages internal accounts",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/core/tree/main/packages/accounts-controller#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/core/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/core.git"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build:docs": "typedoc",
+    "changelog:validate": "../../scripts/validate-changelog.sh @metamask/accounts-controller",
+    "publish:preview": "yarn npm publish --tag preview",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@metamask/base-controller": "^3.2.1",
+    "@metamask/eth-snap-keyring": "^0.2.2",
+    "@metamask/keyring-api": "^0.2.5",
+    "@metamask/keyring-controller": "^7.4.0",
```

Comment:
This one can be a `devDependency`, as we now use just action types?

Reply:
yes, we can move `@metamask/keyring-controller` into devDependency. Changed in 2c2fa52656e82694c8b4e48440e718cb8d3fd86d